tests/baselines/reference/checkMergedGlobalUMDSymbol.errors.txt

```diff
@@ -0,0 +1,22 @@
+tests/cases/compiler/global.d.ts(6,16): error TS2403: Subsequent variable declarations must have the same type.  Variable 'THREE' must be of type 'typeof import("tests/cases/compiler/global")', but here has type 'typeof import("tests/cases/compiler/three")'.
+
+
+==== tests/cases/compiler/three.d.ts (0 errors) ====
+    export namespace THREE {
+      export class Vector2 {}
+    }
+    
+==== tests/cases/compiler/global.d.ts (1 errors) ====
+    import * as _three from './three';
+    
+    export as namespace THREE;
+    
+    declare global {
+      export const THREE: typeof _three;
+                   ~~~~~
+!!! error TS2403: Subsequent variable declarations must have the same type.  Variable 'THREE' must be of type 'typeof import("tests/cases/compiler/global")', but here has type 'typeof import("tests/cases/compiler/three")'.

```

Comment:
Well, this error is not specific to `export as namespace`, making it more confusing. But it is correct. The two declarations of `THREE` are

1. `export as namespace THREE`, which has the type of the containing module, `typeof import("./global")`.
2. `export const THREE`, which is in the global namespace. Its declared type is `typeof _three`, which is the type of the imported module './three', `typeof import("./three")`

These are not the same type, although they might be assignable to each other.